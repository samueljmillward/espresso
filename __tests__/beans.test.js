import React from 'react';
import { shallow } from 'enzyme';
import Beans from '../src/pages/beans';

describe('renders UI', () => {
  it('beans page', () => {
    const wrapper = shallow(<Beans />);
    expect(wrapper).toMatchSnapshot();
  });
});
