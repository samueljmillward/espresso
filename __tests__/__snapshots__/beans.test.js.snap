// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI beans page 1`] = `
<Fragment>
  <Navbar />
  <WithStyles(ForwardRef(Grid))
    className="makeStyles-gridContainer-1"
    container={true}
    justify="center"
    spacing={4}
  />
  <NewBean
    className="makeStyles-addButton-3"
    setBeansList={[Function]}
  />
</Fragment>
`;
